romancal/lib/siafdb.py

```diff
@@ -0,0 +1,302 @@
+"""
+SIAF Database Access.
+
+Provide a common interface to different versions of the SIAF.
+
+Under operations, the SIAF is found in a sqlite database.
+Otherwise, use the standard interface defined by the `pysiaf` package
+"""
+
+import logging
+import os
+from collections import namedtuple
+from datetime import date
+from pathlib import Path
+
+from .basic_utils import LoggingContext
+
+# Setup logging
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+
+# Map instrument three character mnemonic to full name
+INSTRUMENT_MAP = {
+    "fgs": "fgs",
+    "mir": "miri",
+    "nis": "niriss",
+    "nrc": "nircam",
+    "nrs": "nirspec",
+}
```

Comment:
Don't we need Roman instruments here?

Reply:
was holding this for near future development, but it has been removed completely now.